pkg/scanners/clairify/clairify.go

```diff
@@ -30,14 +31,22 @@ import (
 	"github.com/stackrox/scanner/pkg/clairify/client"
 	"github.com/stackrox/scanner/pkg/clairify/types"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/status"
 )
 
 const (
 	// TypeString is the name of the Clairify scanner.
 	TypeString = "clairify"
 
-	clientTimeout             = 5 * time.Minute
+	// defaultClientTimeout default timeout for scanner calls.
+	defaultClientTimeout = 5 * time.Minute
+
+	// nodeScanClientTimeout is used for node scanning operations, it is shorter than
+	// the default because we expect it to have only the database on its path.
+	nodeScanClientTimeout = 10 * time.Second
```

Comment:
We decreased the max duration of this op from 5m to 10s - this is great!
But isn't 10s too little to include the first call and potential retries?

Reply:
Maybe. Are you concerned with retrying calls that can take more than or close to 10s? If yes, I will address this concern in [your other comment](https://github.com/stackrox/stackrox/pull/4971#discussion_r1119083090). If not, please clarify how 10s can be too little for the first call and retries.